Common.mk

```diff
@@ -49,12 +49,11 @@ CODEBUILD_BUILD_IMAGE?=
 CLONE_URL?=$(call GET_CLONE_URL,$(REPO_OWNER),$(REPO))
 #HELM_CLONE_URL=$(call GET_CLONE_URL,$(HELM_SOURCE_OWNER),$(HELM_SOURCE_REPOSITORY))
 HELM_CLONE_URL=https://github.com/$(HELM_SOURCE_OWNER)/$(HELM_SOURCE_REPOSITORY).git
+ARTIFACTS_PATH?=$(MAKE_ROOT)/_output/tar
```

Comment:
If we can avoid changing this file at all that would be best, since we try to sync this between eks-a as well.

Could we just set the ARTIFACTS_PATH in our code build spec?

Reply:
probably, i'll take a look